index.js

```diff
@@ -44,3 +44,291 @@ form.addEventListener('submit', (event) => {
+    image: 'Modal-pix.png',
+    technologies: ['CSS', 'HTML', 'Javascript'],
+    company: 'Canopy',
+    role: 'Back End Dev',
+    year: '2015',
+    liveLink: 'https://coker2019.github.io/Coker_Portfolio/',
+    sourceLink: 'https://github.com/coker2019/Coker_Portfolio',
+  },
+  {
+    title: 'Facebook 360',
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+    image: 'Modal-pix.png',
+    technologies: ['CSS', 'HTML', 'Javascript'],
+    company: 'Canopy',
+    role: 'Back End Dev',
+    year: '2015',
+    liveLink: 'https://coker2019.github.io/Coker_Portfolio/',
+    sourceLink: 'https://github.com/coker2019/Coker_Portfolio',
+  },
+  {
+    title: 'Uber Navigation',
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+    image: 'SnapperModals.png',
+    technologies: ['CSS', 'HTML', 'Javascript'],
+    company: 'Canopy',
+    role: 'Back End Dev',
+    year: '2015',
+    liveLink: 'https://coker2019.github.io/Coker_Portfolio/',
+    sourceLink: 'https://github.com/coker2019/Coker_Portfolio',
+  },
+
+];
+
+const projectLists = () => {
+  projectContainer.innerHTML = '';
+  for (let i = 0; i < worksProjects.length; i += 1) {
+    const projectWrapper = document.createElement('div');
+    projectWrapper.setAttribute('class', 'works main');
+    if (i % 2 !== 0) {
+      projectWrapper.setAttribute('class', 'works main main2');
+    }
+
+    const imageWrapper = document.createElement('div');
+    imageWrapper.setAttribute('class', 'cont1');
+    const projectImage = document.createElement('img');
+    projectImage.setAttribute('src', worksProjects[i].imageURL);
+
+    const contBlock = document.createElement('div');
+    contBlock.setAttribute('class', 'cont2');
+
+    const projectName = document.createElement('h2');
+    projectName.textContent = worksProjects[i].name;
+
+    const listItems2 = document.createElement('ul');
+    const list1 = document.createElement('li');
+    list1.textContent = worksProjects[i].company;
+    list1.setAttribute('class', 'show');
```

Comment:
- [ ] While you make a great effort 😃 I see that when I took a look at the style of the popup I can not see all the content of the popup in the mobile version as it is hidden on the page, so please 💯 try to fix that to let the user when opens the popup be able to see all the content of the popup you can fix that by adding the scroll vertically to let the user be able to see all the content 😊

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Figma design | Your design| 
| --- | --- |
|  ![image](https://user-images.githubusercontent.com/83360636/233742226-1c120615-c066-4945-b3a4-99655a53c092.png)   |  ![image](https://user-images.githubusercontent.com/83360636/233742191-28cfd191-5fa0-4558-bda2-31465120ccce.png)   |